static/scss/pages/settings.scss

```diff
@@ -0,0 +1,120 @@
+#settings-main {
```

Comment:
Blocking request: Could you swap this to a `.settings-main` class? (As they can be super difficult to override, as outlined in the [Protocol coding docs](https://protocol.mozilla.org/docs/css-guide.html#simple-selectors)) I think we should refrain from any ID selector name-spacing when possible.

Reply:
Haha, I was explicitly trying to avoid the styles being overridable/reusable to make sure they don't leak into other parts of the website :P But here: 6925537e48674cb604ccac31773e38473c8651a7